templates/auth_ui_change_password.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+{{ template "auth_ui_html_head.html" . }}
+<body class="page">
+<div class="content">
+
+{{ template "auth_ui_header.html" . }}
+
+{{ template "auth_ui_nav_bar.html" }}
+
+<form class="simple-form vertical-form form-fields-container pane" method="post" novalidate>
+{{ $.CSRFField }}
+
+<h1 class="title primary-txt">{{ template "change-password-page-title" }}</h1>
+
+{{ template "ERROR" . }}
+
+<div class="description primary-txt">{{ template "change-password-page-description" }}</div>
+
+<input
+	id="password"
+	data-password-policy-password=""
+	class="input text-input primary-txt"
+	type="password"
+	autocomplete="new-password"
+	name="x_password"
+	placeholder="{{ template "new-password-placeholder" }}"
+>
+
+<input
```

Comment:
We should also have `autocomplete="new-password"` for password confirmation field.

ref: https://www.chromium.org/developers/design-documents/form-styles-that-chromium-understands

Reply:
If it is auto-completed, that would defeat the purpose of checking typo.

Reply:
Since the autocomplete type is `new-password`, I suppose it would be auto-completed only if the new password is auto-generated?

Reply:
OK. Let's trust chrome.

Reply:
My original concern is if the new password is created by the user, and the confirm is auto-completed, typo cannot be detected.